lib/mock/elixir_auth_google.ex

```diff
@@ -0,0 +1,25 @@
+defmodule AuthMvp.Mock.ElixirAuthGoogle do
+
+  def generate_oauth_url(_conn, _state) do
+    "google oauth url"
+  end
+
+
+  def get_token(_code, _conn) do
+    {:ok, %{:access_token => "token"}}
+  end
+
+  def get_user_profile(_token) do
+    {:ok,
+     %{
+       email: "nelson@gmail.com",
+       email_verified: true,
+       family_name: "Correia",
+       given_name: "Nelson",
+       locale: "en",
+       name: "Nelson Correia",
+       picture: "https://lh3.googleusercontent.com/a-/AAuE7mApnYb260YC1JY7a",
```

Comment:
this picture URL fails. but I'm guessing the purpose of this file is just to pass _automated_ tests. 👍

Reply:
yes it's a dummy data that I copied from elixir_auth_google.

Reply:
Cool. keep going. 👍